posts/2021-09-03-210010-beta.adoc

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+title: "Java 17 beta support, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Java 17 beta support, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.10-beta provides long-awaited Java 17 beta support, offering a lot of new enhancements and functionality. Also included are updates to MicroProfile Context Propagation, meaning that you can now use MicroProfile Context Propagation within your Jakarta EE 9 applications. This beta also offers JWE token format support for OpenID Connect Client and Social Media Login, as well as Jakarta EE 9 support for the remaining Open Liberty value-add features.
```

Comment:
Open Liberty 21.0.0.10-beta provides long-awaited Java 17 beta support, offering a lot of new enhancements and functionality. Also included is JWE token format support for OpenID Connect Client (OIDC) and Social Media Login. This beta also offers updates to MicroProfile Context Propagation, meaning that you can now use MicroProfile Context Propagation within your Jakarta EE 9 applications, as well as Jakarta EE 9 support for the remaining Open Liberty value-add features.